src/main/resources/db/migration/V1_11__prisoner_iep_level.sql

```diff
@@ -0,0 +1,3 @@
+-- Drop `sequence` column (and associated indexes/unique constraints)
```

Comment:
Could give this more descriptive name like `1_11__prisoner_iep_level_drop_sequence.sql`